### DISCUSSION TITLE
trouble with syslog from firewall to Security Onion...(need help)

### DISCUSSION THREAD

**Discussion by DinhDiesel** [Security-Onion-Solutions/securityonion#5756] [Unsupported Versions] [answered]
![syslogissue](https://user-images.githubusercontent.com/87437722/135895856-90ca848a-f665-4354-ba42-8f07b233b37a.JPG)
Trying to dump my firewall syslogs which is set to (514/udp) from firewall console then just changed it to tcp still doesnt work. I originally had syslog forwarding to my heavy, now I have it forwarding to my manager(10.10.10.199). Not sure what i'm missing here....

**Comment by dougburks** [comment]
Did you run `so-allow` to allow the traffic?
https://docs.securityonion.net/en/2.3/so-allow.html

**Reply by DinhDiesel** [reply]
yes i did

**Comment by dougburks** [comment, marked as answer]
In https://github.com/Security-Onion-Solutions/securityonion/discussions/5947 you mention that you just have a manager and heavy node, so I'll copy my reply here as well:

> If your manager is just a manager and not a managersearch, then there may not be anywhere for those logs to go. Heavy nodes only index their own logs and not any logs from the manager. So if you want to send Wazuh logs to the manager it would either need to be a managersearch or you would need a separate search node. For more information, please see:
https://docs.securityonion.net/en/2.3/architecture.html

**Reply by DinhDiesel** [reply]
@dougburks understood thank you!
